packages/multi-select-combo-box/test/chips.test.js

```diff
@@ -205,9 +205,20 @@ describe('chips', () => {
         expect(getChipContent(chips[1])).to.equal('orange');
       });
 
+      it('should move all chips to the overflow if there is not enough space for a single chip', async () => {
+        comboBox.style.width = '100px';
+        await nextResize(comboBox);
+
+        comboBox.selectedItems = ['apple', 'orange'];
+        await nextRender();
+
+        const chips = getChips(comboBox);
+        expect(chips.length).to.equal(1);
+        expect(overflow.hasAttribute('hidden')).to.be.false;
+      });
+
       it('should set show max width on the chip based on CSS property', async () => {
         comboBox.style.width = 'auto';
-        comboBox.clearButtonVisible = true;
```

Comment:
Had to remove this as with clear button all chips are collapsed to the overflow now.